drivers/storage/portworx/component/telemetry.go

```diff
@@ -178,12 +183,18 @@ func (t *telemetry) reconcileCCMGo(
 	cluster *corev1.StorageCluster,
 	ownerRef *metav1.OwnerReference,
 ) error {
-	if err := t.reconcileCCMGoRolesAndRoleBindings(cluster, ownerRef); err != nil {
+	// Remove old ccm components first if exist
+	if err := k8sutil.DeleteConfigMap(t.k8sClient, TelemetryCCMProxyConfigMapName, cluster.Namespace, *ownerRef); err != nil {
 		return err
 	}
-	if cluster.Status.ClusterUID == "" {
-		logrus.Warn("clusterUID is empty, wait for it to fill telemetry proxy config")
-		return nil
+	if err := t.createCCMGoSCCComponents(cluster, ownerRef); err != nil {
```

Comment:
rename this to reconcileCCMGoClusterRolesAndClusterRoleBindings

Reply:
We have a service account here inside as well, all required to support scc

Reply:
Understood, but it's not for SCC only, i was thinking about matching this with role and roleBinding, serviceAccount can be created separately